readers.py

```diff
@@ -21,7 +21,7 @@
 
 import os
 import glob
-import statvfs
+import shutil
```

Comment:
For consistency you should use `os.statvfs`, they're various uses of that in sugar code base and none of `shutil.disk_usage`.

Reply:
Done in `1f1bb36`